crates/datasources/src/excel/table.rs

```diff
@@ -22,7 +25,16 @@ pub async fn excel_streaming_table(
 ) -> Result<Arc<dyn TableProvider>, ExcelError> {
```

Comment:
It wouldn't be absurd to just convert this type into `ExtensionError` or `DataFusionError` at this point rather than at the call site.